eclair-core/src/main/resources/reference.conf

```diff
@@ -90,8 +90,11 @@ eclair {
     dust-limit-satoshis = 546
     max-remote-dust-limit-satoshis = 600
     htlc-minimum-msat = 1
-    // The following parameters apply to each HTLC direction (incoming or outgoing), which means that the total HTLC limits will be twice what is set here
+    // The following parameters apply to each HTLC direction (incoming or outgoing), which means that the total HTLC limits will be twice what is set here.
```

Comment:
This comment is inaccurate (it was already inaccurate before this PR), since the actual value depends on the remote parameter.